ui/tests/integration/components/console/log-json-test.js

```diff
@@ -19,8 +19,7 @@ module('Integration | Component | console/log json', function (hooks) {
     this.set('content', objectContent);
 
     await render(hbs`{{console/log-json content=content}}`);
-    const instance = this.codeMirror.instanceFor(find('[data-test-component=json-editor]').id);
```

Comment:
this old way of selecting the codemirror instance no longer works. I suspect because we're initializing it through a modifier instead of on the component via ivycodemirror.